src/app/profile/overview/spaces/spaces.component.less

```diff
@@ -10,6 +10,9 @@
   overflow-y: auto;
   overflow-x: hidden;
 }
+.no-scroll {
```

Comment:
I'm thinking this might be better served in the `src/assets/stylesheets/shared` directory, so that it can be reused in the future? Might be worth naming it so that it also easily declares that this is for the vertical axis, and not all axes.

Reply:
Good call, I've renamed and moved it out !